<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Four In A Row</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="7" colspan="3">
     <widget class="QPushButton" name="resetButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="accessibleName">
       <string>dropButton1</string>
      </property>
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7" colspan="3">
     <widget class="QPushButton" name="continueButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CONTINUE</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="dropButton1">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: red</string>
      </property>
      <property name="text">
       <string>DROP</string>
      </property>
      <property name="autoRepeatDelay">
       <number>500</number>
      </property>
      <property name="autoRepeatInterval">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="dropButton2">
      <property name="styleSheet">
       <string notr="true">background-color: red</string>
      </property>
      <property name="text">
       <string>DROP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLabel" name="redScore">
      <property name="font">
       <font>
        <pointsize>60</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="dropButton3">
      <property name="styleSheet">
       <string notr="true">background-color: red</string>
      </property>
      <property name="text">
       <string>DROP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="3" colspan="7">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="dropButton5">
      <property name="styleSheet">
       <string notr="true">background-color: red</string>
      </property>
      <property name="text">
       <string>DROP</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="dropButton4">
      <property name="styleSheet">
       <string notr="true">background-color: red</string>
      </property>
      <property name="text">
       <string>DROP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QLabel" name="yellowScore">
      <property name="font">
       <font>
        <pointsize>60</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: yellow</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="dropButton6">
      <property name="styleSheet">
       <string notr="true">background-color: red</string>
      </property>
      <property name="text">
       <string>DROP</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="dropButton7">
      <property name="styleSheet">
       <string notr="true">background-color: red</string>
      </property>
      <property name="text">
       <string>DROP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>60</pointsize>
       </font>
      </property>
      <property name="text">
       <string>:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
